model/src/test/kotlin/ScanRecordTest.kt

```diff
@@ -34,8 +34,7 @@ class ScanRecordTest : StringSpec({
             ---
             scan_results:
               type:namespace:name:version:
-              - provenance:
-                  download_time: "1970-01-01T00:00:00Z"
+              - provenance: []
```

Comment:
Should this be `{}` instead of `[]` (like already in `file-counter-expected-output-for-analyzer-result.yml`)?